Lib/asyncio/streams.py

```diff
@@ -1366,7 +1371,7 @@ def writelines(self, data):
     def _fast_drain(self):
         # The helper tries to use fast-path to return already existing
         # complete future object if underlying transport is not paused
```

Comment:
Unrelated to the PR, but could make a minor grammar fix here? I could make further suggestions to the phrasing of the code comments for this method, but I figured this suggestion would be easier and uncontroversial. 

```suggestion
         # complete future objects if the underlying transport is not paused
```

Reply:
fast drain works with the single object, not multiple ones.

Reply:
> fast drain works with the single object, not multiple ones.

Oops, I'll adjust the suggestion.